test/raven.test.js

```diff
@@ -6,20 +6,13 @@ var sinon = require('sinon');
 var chai = require('chai');
 var assert = chai.assert;
 
-var proxyquire = require('proxyquireify')(require);
-
-var TraceKit = require('../vendor/TraceKit/tracekit');
-
-var _Raven = proxyquire('../src/raven', {
-  // Ensure same TraceKit obj is shared (without specifying this, proxyquire
-  // seems to clone dependencies or something weird)
-  '../vendor/TraceKit/tracekit': TraceKit
-});
```

Comment:
This was probably here for a good reason.

I removed it because it depends on browserify. Also most tests seem to work without it.

Reply:
I'm fine with removing it if all tests are passing on HeadlessChrome/HeadlessFirefox